### PR TITLE
fix: sync, sync before upgrade

### PR DESCRIPTION
**Pull request by divyenduz**

### CODE REVIEW

File: .github/workflows/keep-prisma-dependencies-updated.yaml [hunk 1]
```diff
@@ -59,7 +59,12 @@ jobs:
   report-to-slack-failure:
     if: github.repository_owner == 'prisma' && failure()
     runs-on: ubuntu-latest
-    needs: [check-for-update-latest,  check-for-update-dev,  check-for-update-patch-dev]
+    needs:
+      [
+        check-for-update-latest,
+        check-for-update-dev,
+        check-for-update-patch-dev,
+      ]
```
Comment (steebchen) [inline .github/workflows/keep-prisma-dependencies-updated.yaml]: don't do this, it should be either `['','']` or 
```
needs:
  - 1
  - 2
```

Comment (divyenduz) [reply .github/workflows/keep-prisma-dependencies-updated.yaml]: This was prettier or something. Reverting.

Comment (divyenduz) [reply .github/workflows/keep-prisma-dependencies-updated.yaml]:
```suggestion
    needs:
      [check-for-update-latest, check-for-update-dev, check-for-update-patch-dev]
```

File: .github/workflows/keep-prisma-dependencies-updated.yaml [hunk 2]
```diff
@@ -59,7 +59,8 @@ jobs:
   report-to-slack-failure:
     if: github.repository_owner == 'prisma' && failure()
     runs-on: ubuntu-latest
-    needs: [check-for-update-latest,  check-for-update-dev,  check-for-update-patch-dev]
+    needs:
```
Comment (divyenduz) [inline .github/workflows/keep-prisma-dependencies-updated.yaml]:
```suggestion
    needs: [check-for-update-latest,  check-for-update-dev,  check-for-update-patch-dev]
```

File: .github/workflows/keep-prisma-dependencies-updated.yaml [hunk 3]
```diff
@@ -60,6 +60,7 @@ jobs:
     if: github.repository_owner == 'prisma' && failure()
     runs-on: ubuntu-latest
     needs: [check-for-update-latest,  check-for-update-dev,  check-for-update-patch-dev]
+      [check-for-update-latest, check-for-update-dev, check-for-update-patch-dev]
```
Comment (divyenduz) [inline .github/workflows/keep-prisma-dependencies-updated.yaml]:
```suggestion
```

**Comment by divyenduz** [comment]
Had a chat with Luca and this doesn't work. Internal chat https://prisma-company.slack.com/archives/CUH6B8FUJ/p1601572060019100

**Pull request closed without merging**